package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mesg-js",
-  "version": "4.3.0-beta",
+  "version": "4.3.0-beta.1",
```

Comment:
can you remove the version modification on the PR, this will be done when we release